x-pack/legacy/plugins/siem/public/components/timeline/footer/last_updated.tsx

```diff
@@ -37,46 +33,38 @@ export const Updated = pure<{ date: number; prefix: string; updatedAt: number }>
 
 Updated.displayName = 'Updated';
 
-export class LastUpdatedAt extends React.PureComponent<LastUpdatedAtProps, LastUpdatedAtState> {
-  public readonly state = {
-    date: Date.now(),
-  };
-  private timerID?: NodeJS.Timeout;
+const prefix = ` ${i18n.UPDATED} `;
 
-  public componentDidMount() {
-    this.timerID = setInterval(() => this.tick(), 10000);
-  }
+export const LastUpdatedAt = React.memo<LastUpdatedAtProps>(({ compact = false, updatedAt }) => {
+  const [date, setDate] = useState<number>(Date.now());
+  const timerID = useRef<NodeJS.Timeout | null>(null);
 
-  public componentWillUnmount() {
-    clearInterval(this.timerID!);
+  function tick() {
+    setDate(Date.now());
   }
 
-  public tick() {
-    this.setState({
-      date: Date.now(),
-    });
-  }
+  useEffect(() => {
+    timerID.current = setInterval(() => tick(), 10000);
+    return () => {
+      clearInterval(timerID.current!);
```

Comment:
When you change the type for the timeout to be that of a browser this `current` member should go away since the timerID for browsers aren't the same thing as NodeJS. They don't have a current but rather just an opaque handle.